components/SiteMenu.vue

```diff
@@ -53,74 +53,58 @@ function handleLinkClick () {
 
 const items = [
   {
-    label: 'Mint NFT',
+    label: 'NFT Book',
     links: [
       {
-        label: 'Mint NFT',
+        label: 'Mint New Books',
```

Comment:
```suggestion
        label: 'Create New Books',
```

Reply:
same ui is used for restock though, not sure if any better words

Reply:
`Print` maybe

Reply:
I am following KDP
<img width="201" alt="image" src="https://github.com/user-attachments/assets/4cb55b70-d879-4435-b86a-03423eefb897">